foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/ConfigLoader.java

```diff
@@ -167,8 +170,14 @@ private Set<Property> reconcileConfigs(Set<Property> baseConfigs, Set<Property>
      * @param properties to be written to store
      */
     public void write(Set<Property> properties) {
-        propertyDao.write(properties);
-        logger.info("Successfully wrote all properties to the store.");
+        try {
+                propertyDao.write(properties);
+                logger.info("Successfully wrote all properties to the store.");
+                updateLoadStatus(fullyLoaded);
+            } catch (Exception e) {
+                logger.error("Error updating properties.", e);
+                updateLoadStatus(false);
```

Comment:
Q: should we continue to throw out the exception once we capture the updateLoadStatus failed.

Reply:
In updateLoadStatus, it will throw out exception if upload/write fails. So I think that should be sufficient?